snippets/logging-clusteradmin-access-logs-snip.adoc

```diff
@@ -0,0 +1,37 @@
+// Text snippet included in the following assemblies:
+//
+//
+// Text snippet included in the following modules:
+//
+// * modules/cluster-logging-loki-deploy.adoc
+//
+:_content-type: SNIPPET
+
+[IMPORTANT]
+====
+The error `Parse error: input size too long (XXXX > 5120)` comes when a `cluster-admin` user tries to query application logs for multiple namespaces, where the sum of total characters of all the namespaces in the cluster is more than 5120. To achieve better control over access to logs in LokiStack, the `cluster-admin` user should be a part of the `system:cluster-admins` group. If the group does not exist, create a new group and add the `cluster-admin` users.
+====
+
+== Creating a new group for cluster-admin user role
+
+You can create a new group and add the `cluster-admin` users by using the following commands:
+
+
+. Create a new group.
++
+[source,terminal]
+----
+$ oc adm groups new cluster-admin
+----
+. Add the `cluster-admin` user.
++
+[source,terminal]
+----
+$ oc adm groups add-users cluster-admin username
```

Comment:
I don't believe this is accurate. If I understand correctly, this command won't add a user named `cluster-admin`, it will add `username` to the `cluster-admin` group.

We should also enclose user-replaced values in angle brackets (`<username>`).

Reply:
Fixed this. Thanks!